5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/utils/CommonServices.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Please remove this class as it's not necessary

Reply:
This class contains one common method that is used for login